Twittimation/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+using Twittimation.Commands;
+using Twittimation.IO;
+
+namespace Twittimation
+{
+    public class Program
+    {
+        public static bool Exiting = false;
+        private static AppDataJsonIo _io { get; } = new AppDataJsonIo("Twittimation");
+        public readonly static string TasksFileName = "Tasks";
+        public readonly static string TwitterCredentialsFileName = "Credentials";
+
+        public static void Main(string[] args)
+        {
+            Console.SetIn(new StreamReader(Console.OpenStandardInput(8192)));
+            var cli = new Cli("Invalid Command, type help to display all commands with their help sections");
+            AddCommands(cli);
+            if (args.Length == 0)
```

Comment:
If no command is supplied, the program should execute "RunSchedule"

Reply:
Is that better than having no command allowing you to then input commands without specifying the program again?

Reply:
Yes. It's better because of shell scripts.